tests/UnitTests/Rsp.IrasPortal.UnitTests/Services/ReviewBodyServiceTests/GetReviewBodyById.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Rsp.IrasPortal.Application.DTOs;
+using Rsp.IrasPortal.Application.Responses;
+using Rsp.IrasPortal.Application.ServiceClients;
+using Rsp.IrasPortal.Services;
+
+namespace Rsp.IrasPortal.UnitTests.Services.ReviewBodyServiceTests;
+
+public class GetReviewBodiesByIdTests : TestServiceBase<ReviewBodyService>
+{
+    [Theory]
+    [AutoData]
```

Comment:
Perhaps combine using single line? Same issue in a few other tests across the project.
```suggestion
    [Theory, AutoData]
```